ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/cco-manual-workload-identity/minimal-permission/cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-minimal-permission-workflow.yaml

```diff
@@ -6,7 +6,6 @@ workflow:
     - chain: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-provision
```

Comment:
I guess we can inject `ref: gcp-provision-minimal-permission` to `cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-provision` chain to drop `cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-minimal-permission` install workflow.

Reply:
Sorry, my bad, I shall remove the step `gcp-provision-minimal-permission` from the workflow, because the workflow is with `ENABLE_MIN_PERMISSION_FOR_STS: true`, and CCO in manual mode, while the IPI with minimal permissions testing is for CCO in Mint mode.

Reply:
I meat, now we have two install workflow:
cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-minimal-permission
cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity

The only difference is `gcp-provision-minimal-permission`, so I think we can add `gcp-provision-minimal-permission` to `cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity`, then drop `cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-minimal-permission`.

Reply:
Hold on, let me confirm, thanks!